client/cli/src/params/database_params.rs

```diff
@@ -17,24 +17,24 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::arg_enums::Database;
+use clap::Args;
 use sc_service::TransactionStorageMode;
-use structopt::StructOpt;
 
 /// Parameters for block import.
-#[derive(Debug, StructOpt, Clone)]
+#[derive(Debug, Clone, Args)]
 pub struct DatabaseParams {
 	/// Select database backend to use.
-	#[structopt(
+	#[clap(
 		long,
 		alias = "db",
 		value_name = "DB",
-		case_insensitive = true,
-		possible_values = &Database::variants(),
+		ignore_case = true,
+		possible_values = Database::variants(),
```

Comment:
No replacement to `arg_enum` here?

Reply:
if I use `arg_enum` here, do I need to remove the `variants` method of `Database` and `impl std::str::FromStr for Database`?

Reply:
I have no idea what that means, I'm just pointing out how it looks different from the other enums where you put `arg_num`.

Reply:
Because the `variants` method of other enums are generated by `structopt`, but the  `Database::variants` and `WasmExecutionMethod::variants` are implemented manually.